lms/templates/courseware_search/search_list.underscore

```diff
@@ -9,7 +9,9 @@
 
     <% if (hasMoreResults) { %>
         <a class="search-load-next" href="javascript:void(0);">
-            <%= interpolate(gettext("Load next %s results"), [pageSize]) %>
+            <%= interpolate(
+                gettext("Load next %s results"), [pageSize]
```

Comment:
@louyihua - could I ask for one last change here, which is to make `%s` into a variable name to add better translator context? Also, this should be a call to ngettext. So,

```javascript
<%= interpolate(
    ngettext("Load next %(num_items) result", "Load next %(num_items) results", pageSize),
    { num_items: pageSize },
    true
) %>
```

The reason for this is that some languages have plural forms that are dependant on the final digit in a number, so just because this is always going to be > 2, we can't assume that we can use a generic "plural" form for all languages.

cc @muhammad-ammar : this is a change you made, and something you should be aware of.